src/main/java/org/fundacionjala/coding/dennis/SortInner.java

```diff
@@ -0,0 +1,49 @@
+package org.fundacionjala.coding.dennis;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.StringJoiner;
+
+/**
+ *
+ * @author DennisMont
+ *
+ */
+public final class SortInner {
+
+    public static final int LIMIT = 3;
+
+    /**
+     *
+     */
+    private SortInner() {
+    }
+
+    /**
+     *
+     * @param st cadena
+     * @return cadena
+     */
+    public static String sortInner(final String st) {
+        String[] words = st.split(" ");
+        StringJoiner sj = new StringJoiner(" ");
+        for (String word : words) {
+            sj.add(word.length() > LIMIT ? word.substring(0, 1)
+                    .concat(inverseOrder(word.substring(1, word.length() - 1)))
+                    .concat(word.substring(word.length() - 1)) : word);
+        }
+        return sj.toString();
+    }
+
+    /**
+     *
+     * @param inner cadena
+     * @return cadena
+     */
+    public static String inverseOrder(final String inner) {
+        ArrayList<String> achar = new ArrayList<String>(Arrays.asList(inner.split("")));
+        Collections.sort(achar, Collections.reverseOrder());
+        return achar.toString().replaceAll("[^a-zA-Z]", "");
```

Comment:
This may be refactored like this.
```java
        List<String> achar = Arrays.asList(inner.split(""));
        achar.sort(Collections.reverseOrder());
        return achar.toString().replaceAll("[^a-zA-Z]", "");
```